src/govuk/components/header/header.test.js

```diff
@@ -30,6 +30,13 @@ describe('Header navigation', () => {
         timeout: 1000
       })
     })
+
+    it('does not show the mobile menu button', async () => {
+      await expect(page).toMatchElement('.govuk-js-header-toggle', {
+        visible: false,
```

Comment:
I tried to do the same thing recently and came to the conclusion that I don't think this does what you think it does – setting `visible: false` doesn't mean 'the element must not be visible in order to match'. Instead it means 'the element doesn't _have_ to be visible in order to match'. So I think this would pass even if the element was visible.

I also tried negating it (`expect(page).not.toMatchElement`), but that doesn't work either – see https://github.com/smooth-code/jest-puppeteer/issues/459.

Please do double check and see if you reach the same conclusion! But I ended up getting the computed style instead – see https://github.com/alphagov/govuk-frontend/commit/e6aa87a14302c1fae741b39757c367f2f48a43df for an example.